cmd/branchLengthsMultiFaBed/branchLengthsMultiFaBed_test.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"github.com/vertgenlab/gonomics/exception"
+	"github.com/vertgenlab/gonomics/fileio"
+	"os"
+	"testing"
+)
+
+var BranchLengthsMultiFaBedTests = []struct {
+	Chrom                      string
+	InFaFile                   string
+	InBedFile                  string
+	VelLengthBedFile           string
+	InitialLengthBedFile       string
+	NumUngappedSitesBedFile    string
+	QOutFile                   string
+	SearchSpaceBed             string
+	SearchSpaceProportion      float64
+	UseSnpDistance             bool
+	Verbose                    bool
+	Epsilon                    float64
+	AllowNegative              bool
+	ZeroDistanceWeightConstant float64
+	VelLengthBedExpected       string
+	InitialLengthBedExpected   string
+	NumUngappedSitesExpected   string
+	QoutExpected               string
+	CavalliSforzaQ             bool
+}{
+	{"chr1",
+		"testdata/test.fa",
+		"testdata/test.in.bed",
+		"testdata/test.Vel.bed",
+		"testdata/test.Initial.bed",
+		"testdata/test.UngappedSites.bed",
+		"testdata/qOut.bed",
+		"",
+		0.5,
+		false,
+		false,
+		1e-8,
+		false,
+		1000,
+		"testdata/expected.Vel.bed",
+		"testdata/expected.Initial.bed",
+		"testdata/expected.NumUngapped.bed",
+		"testdata/QoutExpected.bed",
+		false,
+	},
```

Comment:
Considering the number of flag options, it might be good to write a few more test cases to test these flags as well?